.github/workflows/pr_test_when_merged.yml

```diff
@@ -7,6 +7,13 @@ concurrency:
       group: ${{ github.event.pull_request.number }}
       cancel-in-progress: true
 jobs:
+  dump_github_context:
```

Comment:
this is required job?

Reply:
There was an opinion from @platfowner  that it would be better to leave logging for later debugging, so a job for logging was added.

https://github.com/ainblockchain/ain-js/pull/116#issuecomment-1571155312